presets/babycare.yml

```diff
@@ -0,0 +1,147 @@
+name:
+  default: Baby Care amenities preset
+  en: Baby Care amenities  preset
+  version: '0.1'
```

Comment:
The `version`, `author`, `schema` and `description` lines should not be indented.